scripts/release-on-comment.py

```diff
@@ -211,5 +215,32 @@ def main() -> None:
     trigger_release(Path(options.payload), options.token)
 
 
+class Command:
+    """Represents the command which runs to perform actions
+    Provides abstraction over the way commands are run and to
+    capture errors and deal with error codes.
+    This class keeps the errors sticky, meaning a number of
+    commands can be run and it captures the error from the
+    first failing command.
+    """
+    error = None
+    error_code = None
+    failed_command = None
+
+    @staticmethod
+    def run(command):
+        completed_process = run(command, stdout=PIPE, stderr=PIPE)
+        if completed_process.returncode != 0 and not Command.error_code:
+            Command.error = completed_process.stderr.decode()
+            # we might have numbers in command which will fail in join
+            command = [str(each) for each in command]
+            Command.failed_command = " ".join(command)
```

Comment:
Im not sure if we need it for pytest, but in pedantic correctness, shell quoting should be used

Reply:
You mean escaping special characters? Apologies, I'm new to the concept, but I can definitely research and fix it.